integrated_channels/canvas/dev_docs/postman_collections/Canvas.postman_collection.json

```diff
@@ -0,0 +1,445 @@
+						"1",
+						"modules",
+						"1",
+						"items"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "GET course pages",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{canvas_url}}/api/v1/courses/1/pages",
+					"host": [
+						"{{canvas_url}}"
+					],
+					"path": [
+						"api",
+						"v1",
+						"courses",
+						"1",
+						"pages"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "GET content exports",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{canvas_url}}/api/v1/courses/1/content_exports",
+					"host": [
+						"{{canvas_url}}"
+					],
+					"path": [
+						"api",
+						"v1",
+						"courses",
+						"1",
+						"content_exports"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "GET Accounts",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{canvas_url}}/api/v1/accounts",
+					"host": [
+						"{{canvas_url}}"
+					],
+					"path": [
+						"api",
+						"v1",
+						"accounts"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Create a course",
+			"request": {
+				"auth": {
+					"type": "bearer",
+					"bearer": [
+						{
+							"key": "token",
+							"value": "{{access_token}}",
+							"type": "string"
+						}
+					]
+				},
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "course[name]",
+							"value": "binod created a course using form data",
+							"type": "text"
+						}
+					],
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{canvas_url}}/api/v1/accounts/2/courses",
+					"host": [
+						"{{canvas_url}}"
+					],
+					"path": [
+						"api",
+						"v1",
+						"accounts",
+						"2",
+						"courses"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "DELETE a course",
+			"request": {
+				"auth": {
+					"type": "bearer",
+					"bearer": [
+						{
+							"key": "token",
+							"value": "{{access_token}}",
+							"type": "string"
+						}
+					]
+				},
+				"method": "DELETE",
+				"header": [],
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "course[name]",
+							"value": "binod created a course using form data",
+							"type": "text"
+						}
+					],
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{canvas_url}}/api/v1/courses/<<id>>?event=delete",
+					"host": [
+						"{{canvas_url}}"
+					],
+					"path": [
+						"api",
+						"v1",
+						"courses",
+						"<<id>>"
+					],
+					"query": [
+						{
+							"key": "event",
+							"value": "delete"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "GET course settings",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": ""
```

Comment:
Am I wrong or is this basically just a blank request template?

Reply:
hmm will try re exporting it is not empty in postmane for me